motor/rule_applicator.go

```diff
@@ -54,7 +54,7 @@ type ruleContext struct {
 // of ApplyRulesToRuleSet to change, without a huge refactor. The ApplyRulesToRuleSet function only returns a single error also.
 type RuleSetExecution struct {
 	RuleSet           *rulesets.RuleSet             // The RuleSet in which to apply
-	SpecFileName      string                        // The name of the specification file, used to correctly label location
+	SpecFileName      string                        // The path of the specification file, used to correctly label location
```

Comment:
Originally I had renamed this to SpecFilePath to be more clear it's the path, not just the name, but I realized that's a breaking change. So I just updated the docstring instead.